meta-iot2000-example/recipes-tools/setledcolor/files/setledcolor.py

```diff
@@ -1,50 +1,48 @@
 #!/usr/bin/python
 
-
 import mraa
 import sys
 import os
 
+
 def redled(state):
- redLedFile = "/sys/class/leds/mpio_uart_led:red:user/brightness"
- if(not os.path.isfile(redLedFile)):
-  print("Red LED not available")
-  return
- file = open(redLedFile,'w')
- if(1==state):
-  file.write("1")
- else:
-  file.write("0")
- file.close()
+    redLedFile = "/sys/class/leds/mpio_uart_led:red:user/brightness"
+    if not os.path.isfile(redLedFile):
+        print("Red LED not available")
+        return
+    file = open(redLedFile, 'w')
+    if state == 1:
+        file.write("1")
+    else:
+        file.write("0")
+        file.close()
```

Comment:
Well, this closes the file just when writing `0` to it. `file.close()` should be "de-indented".